tests/v2/validation/registries/config.go

```diff
@@ -1,7 +1,18 @@
 package registries
 
-const RegistriesConfigKey = "registries"
+const (
+	RegistriesConfigKey 	      = "registries"
+	ExistingAuthRegistryConfigKey =  "existingAuthRegistryConfig"
+)
+
+type ExistingAuthRegistryConfig struct {
+	Username string `json:"username" yaml:"username" default:""`
+	Password string `json:"password" yaml:"password" default:""`
+	URL      string `json:"url" yaml:"url" default:""`
+}
 
 type Registries struct {
-	RegistryConfigNames []string `json:"registryConfigNames" yaml:"registryConfigNames" default:"[]"`
+	RegistryConfigNames      []string `json:"registryConfigNames" yaml:"registryConfigNames" default:"[]"`
+	ExistingNoAuthRegistryURL string `json:"existingNoAuthRegistry" yaml:"existingNoAuthRegistry" default:""`
+	ExistingAuthRegistryInfo  *ExistingAuthRegistryConfig `json:"existingAuthRegistry" yaml:"existingAuthRegistry" default:""`
```

Comment:
I think we don't need the `ExistingAuthRegistryConfigKey` constant above. Since the `ExistingAuthRegistryConfig` structure is used here, the key to this struct gonna be `existingAuthRegistry`, but not from `ExistingAuthRegistryConfigKey`; from the defined tag here `json:"existingAuthRegistry" yaml:"existingAuthRegistry" default:"`

Reply:
Thanks for the feedback Ugur, I'll make the changes and test it out. 🙏🏽

Reply:
agreed